x/clp/client/cli/tx.go

```diff
@@ -44,6 +45,41 @@ func TestTxCmd(cdc *wire.Codec) *cobra.Command {
 	}
 }
 
+// create new clp transaction
+func CreateTxCmd(cdc *wire.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "create <ticker> <name> <reserve_ratio>",
+		Short: "Create a token with CLP",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.NewCoreContextFromViper().WithDecoder(authcmd.GetAccountDecoder(cdc))
+
+			// get the from address from the name flag
+			from, err := ctx.GetFromAddress()
+			if err != nil {
+				return err
+			}
+
+			// create the message
+			ticker := args[0]
+			name := args[1]
+			reserveRatio, err := strconv.Atoi(args[2])
```

Comment:
Hm, well i've been putting error handling at the keeper/logic level, rather than at the cli/api level, and so the error should be surfaced from there.

Maybe ideally handling errors at both levels is best, but i think this is fine for now?